content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -39,13 +43,12 @@ All of these questions, among others, represent independent dimensions of config
 All combinations are possible, although some are more common than others and
 some are clearly not very interesting (for example, multiple mesh in a single cluster).
 
-In a production deployment involving multiple clusters, the deployment may use a
-mix of patterns. For example, having more than one control plane is recommended for HA,
-but you could achieve this for a 3 cluster deployment by deploying 2 clusters with
-a single shared control plane and then adding the third cluster with a second
-control plane in a different network. All three clusters could then be configured
-to share both control planes so that all the clusters have 2 sources of control
-to ensure HA.
+In a production environment involving multiple clusters, you can use a mix
+of deployment models. For example, having more than one control plane is
+recommended for HA.
+
+Multiple clusters in the same region can share a control plane in a zonal or
```

Comment:
This seems like it should be part of the previous paragraph?

Reply:
I ended the previous paragraph after the example for emphasis, both of the example and this note.
IMO, this is very important information for readers and I wanted to make sure it was not "hidden" in the paragraph above.